cmake/ProjectsTagsUnstable.cmake

```diff
@@ -33,4 +33,4 @@ set_tag(OsqpEigen_TAG master)
 set_tag(YARP_telemetry_TAG master)
 set_tag(gym-ignition_TAG master)
 set_tag(walking-teleoperation_TAG devel)
-
+et_tag(speech_TAG fix/40)
```

Comment:
```suggestion
set_tag(speech_TAG fix/40)
```

Reply:
Thanks, this is not necessary anymore after https://github.com/robotology/speech/pull/41 .